js/components/controlBar.js

```diff
@@ -753,6 +797,11 @@ var ControlBar = React.createClass({
         continue;
       }
 
+      if (defaultItems[k].name === "multiAudio" &&
```

Comment:
What `k` it is? Rename please

Reply:
I think it would be better to create new task to rename old variables.